tests/lib/fakeinstaller/mk-classic-rootfs.sh

```diff
@@ -79,7 +79,10 @@ fi
 
 # extract the base
 if [ -f /cdrom/casper/base.squashfs ]; then
-    sudo unsquashfs -d "$DST" /cdrom/casper/base.squashfs
+    sudo unsquashfs -f -d "$DST" /cdrom/casper/base.squashfs
+    # TODO: do more here, follow what snapd.postinst purge does
+    rm -rf "$DST"/snap/* "$DST"/var/snap/*
+    rm -f  "$DST"/var/lib/snapd/state.json
```

Comment:
this is ok for the alpha, but there is so much more to remove, for example snap mount units. We really need to understand what is off with the squashfs building

Reply:
+1 on this, approving meanwhile

Reply:
the hopefully easily available most comprehensive thing we could try is:

/usr/lib/snapd/snap-preseed --reset "$DST"